docs/APIRef.Artifacts.md

```diff
@@ -13,7 +20,7 @@ Artifacts are disabled by default. To enable them, specify via **launch argument
 * To record `.dtxrec` (Detox Instruments recordings) for each test, add `--record-performance all`. To open those recordings, you'll need [Detox Instruments](https://github.com/wix/DetoxInstruments). **NOTE:** only iOS is supported.
 * To capture `.uihierarchy` snapshots (**iOS only, Xcode 12.0+**) on view action failures, add `--capture-view-hierarchy enabled`.
 * To take `.png` screenshots before and after each test, add `--take-screenshots all` (or `--take-screenshots failing`, if you want to keep only screenshots of failing tests).  
-Alternatively, you might leverage the [device.takeScreenshot()](APIRef.DeviceObjectAPI.md#devicetakescreenshotname) API for manual control.
+Alternatively, you might leverage the [device.takeScreenshot()](https://wix.github.io/docs/api/artifacts#devicetakescreenshotname) API for manual control.
```

Comment:
@asafkorem, I am strongly against replacing the internal links with the external ones here in the documentation folder. That will make harder the whole process of linting the docs and finding broken anchors inside a specific Git revision. Please don't. I wholeheartedly ask you.

Reply:
Done as discussed.